pkg/sideinput/examples/simple-sideinput/udf/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"path"
+
+	"github.com/fsnotify/fsnotify"
+
+	"github.com/numaproj/numaflow-go/pkg/mapper"
+)
+
+var sideInputPath = "/var/numaflow/side-inputs"
```

Comment:
do they have to know this path, can't they get it from ENV var?

Reply:
We don't expose it as discussed with @whynowy yesterday!

Reply:
Exposing thru ENV might be better, even though they need to memorize something anyways, but it gives us flexibility to have a chance to change the path in the future.

Reply:
or wrap in SDK, `GetSideInputDir()`?

Reply:
Added to shared sdk as `shared.SideInputDir`